website/index.qmd

```diff
@@ -19,17 +18,20 @@ listing:
         image: "../images/bearing_by_latitude_eye.svg"
     type: grid
     grid-columns: 3
-    image-height: 240px
+    image-height: 220px
 ---
+:::column-screen
+:::{class="snow-container" id="snow-container"}
+:::
+:::
 
-<div class="snow-container" id="snow-container"></div>
-<div class="text-overlay">
+:::{class="text-overlay"}
 
 ## OpenSkiStats
 
 an open resource for geospatial metrics of downhill ski areas
 
-</div>
+:::
```

Comment:
since we have divs within divs, it's clearest if we use different number of colons to denote levels, e.g. `::::`

Reply:
will address post merge